argus/backend/service/jenkins_service.py

```diff
@@ -220,7 +220,9 @@ def clone_build_job(self, build_id: str, params: dict[str, str]):
     def build_job(self, build_id: str, params: dict, user_override: str = None):
         queue_number = self._jenkins.build_job(build_id, {
             **params,
-            self.RESERVED_PARAMETER_NAME: g.user.username if not user_override else user_override
+            # use the user's email as the default value for the requested by user parameter,
+            # so it would align with how SCT default works, on runs not trigger by argus
+            self.RESERVED_PARAMETER_NAME: g.user.email.split('@')[0] if not user_override else user_override
```

Comment:
I was actually under impression that it's one or the other,  has something changed?

Reply:
e.g. I'm pretty sure runs started by me from jenkins use my github username in the email reports

Reply:
for me it was roydahan instead of roy

Reply:
@k0machi notting change, it was wrong assumption to begin with, SCT was using email for long long time, this code was using the github login name, which for lots of people isn't the same.